connector_helpscout/models/helpscout_tag/importer.py

```diff
@@ -23,17 +23,13 @@ class HelpScoutTagImportMapper(Component):
 
     direct = [('tag', 'name'),
               ('slug', 'helpscout_slug'),
-              ('created_at', 'backend_date_created'),
-              ('modified_at', 'backend_date_modified'),
               ]
 
     @mapping
     @only_create
     def odoo_id(self, record):
         # Searches project.tags records for matching name
-        tag = self.env['project.tags'].search([
-            ('name', '=', record.tag),
-        ])
+        tag = self.env['project.tags'].search([('name', '=', record.tag)])
```

Comment:
Do we know if name is unique? If not, either limit search to one or do `tag[:1].id` in the return. The limit is better though because less data is being tossed around.